### PR TITLE
AFP-207: Add dummy project which uses typescript and webpack

### PR DESCRIPTION
**Pull request by dpisani-atl**

### CODE REVIEW

File: packages/website/dummy-data/typescript-project/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  mode: 'development',
+  resolve: {
+    // Add `.tsx` as a resolvable extension.
+    extensions: ['.tsx', '.js'],
+  },
+  module: {
+    rules: [
+      { test: /\.tsx?$/, loader: 'babel-loader!ts-loader' },
```
Comment (Noviny) [inline packages/website/dummy-data/typescript-project/webpack.config.js]: Pls can we use `@babel/preset-typescript`, or make an example with it.

File: packages/website/package.json [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+    "description": "The Brisk website documenter",
     "main": "index.js",
     "license": "MIT",
     "private": true,
     "scripts": {
         "test": "jest",
         "dev": "node bin/index.js start --config=dummy-data/simple-project/docs.config.js",
-        "dev:webpack": "node bin/bin.js start --config=dummy-data/custom-webpack-project/docs.config.js",
+        "dev:webpack": "node bin/index.js start --config=dummy-data/custom-webpack-project/docs.config.js",
+        "dev:typescript": "node bin/index.js start --config=dummy-data/typescript-project/docs.config.js",
```
Comment (Noviny) [inline packages/website/package.json]: What's with the choice for these to be in dummy-data not fixtures?

Followup point: If we write fixtures, should we also be okay running against them? Is there a clear ontological difference between them?

Comment (dpisani-atl) [reply packages/website/package.json]: Tbh they should probably be written as tests and not even be put in dummy_data at all

File: packages/website/dummy-data/typescript-project/webpack.config.js (hunk 1 above)
Comment (dpisani-atl) [reply packages/website/dummy-data/typescript-project/webpack.config.js]: I think i'll leave this to test that we can work with projects using the actual typescript server. We have a custom babel test which should cover the babel-typescript use case

**Pull request merged**